### DISCUSSION TITLE
Welcome to TGDates Discussions!

### DISCUSSION THREAD

**Discussion by harshil21** [harshil21/TGDates#1] [Announcements]
## 👋 Welcome!

If you have any ideas on how to improve this web app or just want to share what you've made with this, please feel free to share them here!

**Comment by TheEnterpriseDeveloper** [comment]
Live demo & endpoint don't work. Could you fix it, please?

**Reply by harshil21** [reply]
Unfortunately, it seems that Replit has [deprecated the free static hosting](https://docs.replit.com/hosting/hosting-web-pages) plan. This means I'll not be able to provide the Live Demo nor the endpoint anymore. I'll look for alternative solutions, but until then you'll have to manually host it by following the [steps in the Readme](https://github.com/harshil21/TGDates?tab=readme-ov-file#self-hosting)
